config.txt

```diff
@@ -38,6 +38,9 @@ dtparam=audio=on
 #       this won't work.
 #dtoverlay=w1-gpio-pullup,gpiopin=4
 
+# Support USB gadget mode on the USB-C port
+dtoverlay=dwc2
```

Comment:
ok, but why?  Is this for debugging ?  On PLT-300A, the Type-C is connected do a dumb power port of the PLT-300A MB.